pkg/controller/mizar/builtins.proto

```diff
@@ -52,6 +52,7 @@ message BuiltinsPodMessage {
   string arktos_network = 6;
   string phase = 7;
   repeated InterfacesMessage interfaces = 8;
+  map[string]string annotations = 9;
```

Comment:
not sure whether this is valid grpc protocol type. please confirm.

Reply:
Already talked with Hong and Phu last month, they thought it was correct.

Reply:
Send an email to confirm again.

Reply:
, perUse "map<string, string> annotations = 9;" instead, per Phu confirmation.